src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -28,8 +28,15 @@ export const getAnalyticsSessionId = () => {
   return window.sessionStorage.getItem(INTEGRATIONS_ANALYTICS_SESSION_KEY);
 };
 
-export const isIntegrationDirectoryActive = () =>
-  localStorage.getItem(USE_INTEGRATION_DIRECTORY) === '1';
+export const isIntegrationDirectoryActive = variant => {
+  if (localStorage.getItem(USE_INTEGRATION_DIRECTORY) === '1') {
+    return true;
+  } else if (localStorage.getItem(USE_INTEGRATION_DIRECTORY) === '0') {
+    return false;
+  } else {
+    return variant && variant === '1';
```

Comment:
@NisanthanNanthakumar Pretty sure `return variant === '1';` would work here as well but ok

Reply:
@scefali since I'm using optional chaining to get the variant, it could potentially be undefined.